### DISCUSSION TITLE
Swipe up gesture to mark ‘read all’

### DISCUSSION THREAD

**Discussion by grumpykaiju** [jocmp/capyreader#530] [Feature Requests]
Hello, 

first of all, very cool app!

Sorry if this idea already existed, I couldn't find it.

I often use the RSS reader to browse the article list with one hand. Here the button in the top right corner is difficult to reach. 

So the idea would be a new ‘swipe up gesture’ from the bottom of the article list that sets all items to read.

Once you have performed the gesture, a prompt asking whether you really want to set everything to read would ensure that the gesture was not performed unintentionally.

![swipe_up_gesture](https://github.com/user-attachments/assets/af74d30d-2a7c-468e-93ce-520580778235)

Optionally, the app could switch to the next category when I set the item list of the current list to read. Similar to issues 295 I assume

br daniel

**Comment by jocmp** [comment]
hey daniel. Thanks for the idea and the details. No promises on timeline but I'll add this via #539. As you've seen re: #295 I haven't spent much time on pagination between feeds, so it's something I'd like to improve in the future with features like this.

**Comment by jayemar** [comment]
It would also be nice if this swipe action did another refresh after first marking all of the existing articles as read
